src/main/webapp/app/shared/subject/subject-pair-dialog.component.html

```diff
@@ -26,15 +26,15 @@
                        [(ngModel)]="subject.externalId" [disabled]="subject.id" >
             </div>
 
-            <div *ngIf="subject.personName" >
+            <div *ngIf="!(siteSettings$ | async).hiddenSubjectFields.includes('person_name') && subject.personName" >
```

Comment:
Did you check for a null state? e.g. if siteSettings is null to undefined?

Reply:
as discussed f2f: sitesettings is initialized as an empty list during startup. If the get request goes wrong it is because the backend is failing as a whole which will be caught elsewhere, so imo no explicit error-handling is needed here.